argus/backends/local/local_backend.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 Cloudbase Solutions Srl
```

Comment:
- adds the possibility to use an already existing Windows
      machine that is running, has git installed,
      http winrm enabled and network connectivity.
    - with the NO_SERVICE metadata service we test that pre-metadata
      plugins have been successfully executed and that Cloudbase-Init does not error out.